cmd/helm/repo_update_test.go

```diff
@@ -132,3 +132,29 @@ func TestUpdateCmdStrictFlag(t *testing.T) {
 		t.Errorf("Expected 'Unable to get an update', got %q", got)
 	}
 }
+
```

Comment:
Do you mind adding a test that tests for a successful update with a single repo name?

Reply:
@hickeyma added.